adapter/pkg/synchronizer/apis_fetcher.go

```diff
@@ -87,8 +101,23 @@ func FetchAPIs(id *string, gwLabel []string, c chan SyncAPIResponse, serviceURL
 		Transport: tr,
 	}
 
+	// Populating the payload body with API UUID list
+	if apiUUIDList != nil {
+		body := postData{
+			Uuids: apiUUIDList,
+		}
+		bodyJSON, err = json.Marshal(body)
+		if err != nil {
+			logger.LoggerSync.Errorf("Error marshaling the uuid List: %v", err)
+		}
+	}
+
 	// Create a HTTP request
-	req, err := http.NewRequest("GET", serviceURL, nil)
+	if apiUUIDList == nil {
+		req, _ = http.NewRequest("GET", serviceURL, nil)
```

Comment:
Is handling the error explicitly required here?

Reply:
Fixed with 333d6f625503f297f558519d3126e0b390a70175